samples/sample_multi_transcode/src/pipeline_transcode.cpp

```diff
@@ -46,7 +46,7 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 
 using namespace TranscodingSample;
 
-mfxU32 MFX_STDCALL TranscodingSample::TranscodeRoutine(void   *pObj)
+void MFX_STDCALL TranscodingSample::TranscodeRoutine(void   *pObj)
```

Comment:
I believe we should be able to change this function from C-style to C++ member of ThreadTranscodeContext and remove some explicit casts....

Reply:
Fixed explicit cast. Not sure about to make this function as a member (in this case it needs to make the function to be a static)

Reply:
No, you should be able to bypass static via lambdas and `this` capturing. That's not exactly what you need, but here is an idea:
`[this](void* p){ TranscodeRoutine(p); }`
or something similar should make a trick. I think @mgonchar will be able to guide.

Reply:
OK, seems I got it. Done